File: java/163246.java
/**
 * Lời giải từ bạn TIEU_MAN
 */

int waitingDays(int[] a)
{
    int n = a.length;
    Arrays.sort(a);
    int ans = 0;
    for(int i = n - 1; i > 0; i -= 2) ans += (a[i] - a[i - 1]);
    return ans;
}

File: js/163246.js
/**
 * Lời giải từ bạn fsoft_namnd28
 */

function waitingDays(a) {
	let sum = 0;
	a = a.sort((a, b) => a-b);	
	for (let i = 0; i < a.length; i = i+ 2) {
		sum = sum + ( a[i+1] - a[i]);
	}
	return sum;
}